component/alertrules.libsonnet

```
@@ -36,7 +36,7 @@ local patch_alerts = {
 };
 
 local loadFile(file) =
-  local fpath = 'openshift4-logging/manifests/%s/%s' % [ params.alerts, file ];
+  local fpath = 'openshift4-logging/component/extracted_alerts/%s/%s' % [ params.alerts, file ];
```

This should work without the `openshift4-logging/component` prefix, since the directory of the file should be in the search path (otherwise intra-component includes wouldn't work they way we're doing them):

```suggestion
  local fpath = 'extracted_alerts/%s/%s' % [ params.alerts, file ];
```